File: spec/lua/test_imports_rel_1.lua
local luaunit = require("luaunit")

require("imports_rel_1")

TestImportsRel1 = {}

function TestImportsRel1:test_imports_rel1()
    local r = ImportsRel1:from_file("src/fixed_struct.bin")

    luaunit.assertEquals(r.one, 0x50)
    luaunit.assertEquals(r.two.one, 0x41)
    luaunit.assertEquals(r.two.two.one, 0x43)
end

File: spec/python/test_imports_rel_1.py
import unittest

from imports_rel_1 import ImportsRel1

class TestImportsRel1(unittest.TestCase):
    def test_imports_rel_1(self):
        r = ImportsRel1.from_file("src/fixed_struct.bin")

        self.assertEqual(r.one, 0x50)
        self.assertEqual(r.two.one, 0x41)
        self.assertEqual(r.two.two.one, 0x43)
